core/src/components/cat-input/input-type.ts

```diff
@@ -0,0 +1,21 @@
+export type InputType =
+  | 'button'
+  | 'color'
+  | 'date'
+  | 'datetime-local'
+  | 'email'
+  | 'file'
+  | 'hidden'
```

Comment:
I don't think hidden makes sense, as it will still render the web component. So I think we should remove that. Did you give the others a quick try to see if they are working?

Reply:
Ok, yes, I did a quick check in the index with each one, there are some that look a bit strange but they all work, I will make a push with the different inputs in the index